### DISCUSSION TITLE
Enhancement: Select an index ( Azure AI Search ) from a list.

### DISCUSSION THREAD

**Discussion by shota1998** [danny-avila/LibreChat#2725] [Feature Requests & Suggestions]
### What features would you like to see added?

Users can select an index from a list when using Azure AI Search.

### More details

Current Situation:
Users have to reinstall the Azure AI Search plugin or update the .env file when they want to switch indexes.

Idea:
Allow users to switch indexes through the user interface without the need to reinstall the plugin or modify the .env file.

### Which components are impacted by your request?

UI, Plugins

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I'm planning on general improvements to tools where specific options can be selected for a tool run to allow for this.

It may not be azure ai search-specific but may help your case.
